content/rdi/quickstart/write-behind-guide.md

```diff
@@ -10,15 +10,17 @@ draft:
 hidden: false
 ---
 
-This guide takes you through the creation of a write-behind pipeline.
+This guide takes you through the creation of a write-behind and readh-through pipeline.
 
-> Note: Write Behind is currently in Preview.
+> Note: Write Behind & Read-Through are currently in Preview.
 
 ## Concepts
 
 **Write-behind**: An RDI policy and pipeline to synchronize the data in a Redis DB with some downstream data store.
 You can think about it as a pipeline that starts with change data capture (CDC) events for a Redis database and then filters, transforms, and maps the data to the target data store data structure.
 
+**Read-through**: An RDI policy and pipeline to automatically retrieve data from a downstream data store back to Redis cache in case of cache-miss. The read-through will be triggered by a client trying to get a key from Redis where this key doesn't exist. RDI will retrieve the key, write it to cache and return a response to the client in a sync flow.
```

Comment:
consider this:
A policy and pipeline are implemented in RDI to automatically fetch data from a downstream data store and store it back in the Redis cache when a cache miss occurs. This read-through process is initiated when a client attempts to retrieve a key from Redis that does not exist. RDI will retrieve the key, store it in the cache, and then provide a synchronous response to the client.

Reply:
done